lib/services/all_question_service.dart

```diff
@@ -1,50 +1,34 @@
 import 'package:project_eureka_flutter/models/question_model.dart';
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
 
 class AllQuestionService {
-  // Here Eureka will get data from the database.
-  // At this point it contains just dummy lists of questions for test purposes
-  Future<List> getQuestions() async {
-    List data = [];
-    // List of 2 dummy questions
-    List list1 = List.generate(2, (index) => "Message")
-        .map((val) => QuestionModel(
-              category: "Technology",
-              date: DateTime.now(),
-              status: 1,
-              visible: 1,
-              title: "Linux installation issue - usb not found",
-              description:
-                  "Hello. I have a Smartbuy 16Gb USB 2.0 flash drive (with new memory controller) that is not recognized in any Linux system, but on Windows it recognized and worked fine. When I connect it to the PC on Linux system, nothing happens",
-            ))
-        .toList();
 
-    // List of another 2 dummy questions (different)
-    List list2 = List.generate(2, (index) => "Message")
-        .map((val) => QuestionModel(
-              category: "Household",
-              date: DateTime.now(),
-              status: 0,
-              visible: 1,
-              title: "Windows Issue",
-              description: "Hello",
-            ))
-        .toList();
+  // GET
+  Future<List<QuestionModel>> getQuestions() async {
+    final response = await http.get(Uri.http('IP ADDRESS', '/v1/questions'));
```

Comment:
Please consider looking for a way we can use `.env` file and pass in our ip that way.